util/src/main/java/io/kubernetes/client/util/Strings.java

```diff
@@ -14,10 +14,14 @@
 
 import javax.annotation.Nullable;
 
-public class Strings {
+public final class Strings {
+
+  private Strings() {
+    throw new AssertionError("not supported");
+  }
 
   public static boolean isNullOrEmpty(String value) {
-    return value == null || value.length() == 0;
+    return value == null || value.isEmpty();
```

Comment:
2. same thing, but using a dedicated method. why, though? because `java.lang.String` is highly optimized, you never know when `isEmpty` is going to become an intrinsic. delegating to such methods, is always a good idea.